### PR TITLE
Marked thread capabilities as @deprecated

### PR DESCRIPTION
**Pull request by mreiche**
# Description

Thread capabilites are deprecated and will be removed in future releases.

### CODE REVIEW

File: driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilities.java [hunk 1]
```diff
@@ -66,6 +66,7 @@ static void addGlobalCapability(String key, Object value) {
         GLOBALCAPABILITIES.put(key, value);
     }
 
+    @Deprecated
     static void addThreadCapability(String key, Object value) {
```
Comment (martingrossmann) [inline driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilities.java:70]: You should add a note to use DesktopWebdriverRequset instead.

File: driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManager.java [hunk 2]
```diff
@@ -119,6 +120,7 @@ private static void addGlobalCapability(String key, Object value) {
      * @param key   .
      * @param value .
      */
+    @Deprecated
```
Comment (martingrossmann) [inline driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManager.java:123]: Same like comment for WebDriver Capabilities

File: driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilities.java (hunk 1 above)
Comment (mreiche) [reply driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilities.java:70]: I made it package private instead.

File: driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManager.java [hunk 3]
```diff
@@ -119,6 +120,7 @@ private static void addGlobalCapability(String key, Object value) {
      *
      * @param key   .
      * @param value .
+     * @deprecated Use {@link AbstractWebDriverRequest#getDesiredCapabilities()} instead
```
Comment (martingrossmann) [inline driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverManager.java:123]: Hm... a get-Method cannot be an alternative for add. 

And the returned `DesiredCapabilities` object does not have a practical method to add custom caps, only `merge`...

I think we need a useful method in `AbstractWebDriverRequest`

**Review by martingrossmann — APPROVED** [review]

**Review by erickubenka — APPROVED** [review]

**Pull request merged**